rfhub2/api/endpoints/keywords.py

```diff
@@ -32,9 +39,14 @@ def get_keywords_with_stats(
     limit: int = 100,
     pattern: str = None,
     use_doc: bool = True,
+    collection_id: Optional[int] = None,
 ):
     return repository.get_all_with_stats(
-        skip=skip, limit=limit, pattern=pattern, use_doc=use_doc
+        skip=skip,
```

Comment:
Sorry for not noticing it earlier. Due to different endpoint app is showing only first 100 keywords. Maybe when collection_id is not None we could skip the limit, to have same experience as previosly?

Reply:
good point, I will set UI to use large limit for now